### PR TITLE
Added SendMessageForm

### PR DESCRIPTION
**Pull request by Aswini-D3**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/app.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const app = express();
 
 app.use(express.static(buildPath));
 
-app.use(authenticate);
+//app.use(authenticate);
```
Comment (tpetrina) [inline src/server/app.js:25]: Do not commit this line.

File: src/client/components/MessageForm/SendMessageForm.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
+/*import React from 'react';
```
Comment (tpetrina) [inline src/client/components/MessageForm/SendMessageForm.js:1]: Do not commit commented code.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Please undo the changes in `nodemon.json` file.

File: src/client/App.js [hunk 3]
```diff
@@ -35,6 +36,7 @@ function App() {
           </AuthenticatedRoute>
         </Switch>
       </Router>
+      <SendMessageForm />
```
Comment (benna100) [inline src/client/App.js:39]: The task was not to apply the message form but just to create it for future use

Comment (benna100) [reply src/client/App.js:39]: So please remove this

File: src/client/components/MessageForm/SendMessageForm.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  bottom: 0;
+  width: 100%;
+}
+footer input {
+  margin-left: 10px;
+  width: 1000px;
+}
+footer img {
+  width: 30px;
+}
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.css:15]: The message form is not a footer, just a form

File: src/client/components/MessageForm/SendMessageForm.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import './SendMessageForm.css';
+
+function SendMessageForm() {
+  const [input, setInput] = useState('');
+  const data = {
+    channel_id: null,
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.js:7]: Fix these warnings please

File: src/client/components/MessageForm/SendMessageForm.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import './SendMessageForm.css';
+
+function SendMessageForm() {
+  const [input, setInput] = useState('');
+  const data = {
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.js:6]: Avoid vague variable names

File: src/client/components/MessageForm/SendMessageForm.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    })
+      .then((res) => res.json())
+      .then((messages) => console.log(messages));
+  };
+
+  return (
+    <footer>
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.js:24]: This is not a footer. I would just do it as the form. So the form is the wrapping element

**Review by benna100 — CHANGES_REQUESTED** [review]
Looks good Aswini :) A few comments. Remember especially to make sure you dont commit code not relevant to you story, like fx the nodemon.json file

File: src/client/App.js (hunk 3 above)
Comment (Aswini-D3) [reply src/client/App.js]: Yes I removed.Thank you.

File: src/client/components/MessageForm/SendMessageForm.css (hunk 4 above)
Comment (Aswini-D3) [reply src/client/components/MessageForm/SendMessageForm.css]: I used as a form.Thank you.

File: src/client/components/MessageForm/SendMessageForm.js (hunk 5 above)
Comment (Aswini-D3) [reply src/client/components/MessageForm/SendMessageForm.js]: I fixed.Thank you.

File: src/client/components/MessageForm/SendMessageForm.js (hunk 6 above)
Comment (Aswini-D3) [reply src/client/components/MessageForm/SendMessageForm.js]: I given another name.Thank you.

File: src/client/components/MessageForm/SendMessageForm.js (hunk 7 above)
Comment (Aswini-D3) [reply src/client/components/MessageForm/SendMessageForm.js]: yes i used as a form.Thank you.

**Comment by benna100** [comment]
Great improvements aswini! You are still deleting the nodemon.json file. Please undo that, we dont want to remove that file!! Its essential for running the project.

If you dont know how. 

`git checkout -- nodemon.json`
Whne you do git status you should see that this file has been added. Now just add, commit and push

**Review by benna100 — APPROVED** [review]

**Pull request merged**